x-pack/test_serverless/functional/test_suites/common/discover/x_pack/reporting.ts

```diff
@@ -59,26 +61,30 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   };
 
   describe('Discover CSV Export', function () {
-    // see details: https://github.com/elastic/kibana/issues/197957
-    this.tags(['failsOnMKI']);
-    describe('Check Available', () => {
-      before(async () => {
-        await PageObjects.svlCommonPage.loginAsAdmin();
-        // TODO: emptyKibanaIndex fails in Serverless with
-        // "index_not_found_exception: no such index [.kibana_ingest]",
-        // so it was switched to `savedObjects.cleanStandardList()`
-        await kibanaServer.savedObjects.cleanStandardList();
-        await reportingAPI.initEcommerce();
-        await PageObjects.common.navigateToApp('discover');
-        await PageObjects.discover.selectIndexPattern('ecommerce');
+    before(async () => {
+      await PageObjects.svlCommonPage.loginAsAdmin();
+      // TODO: emptyKibanaIndex fails in Serverless with
+      // "index_not_found_exception: no such index [.kibana_ingest]",
+      // so it was switched to `savedObjects.cleanStandardList()`
+      await kibanaServer.savedObjects.cleanStandardList();
+      await reportingAPI.initEcommerce({
```

Comment:
I've reduced the `initEcommerce` usage to just one. There's no reason why to ingest and delete those docs several time. Also due to those changes we can ingest with concurrency